src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,36 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(actor);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException(
+                    String.format("Can't add actor %s to DB", actor), e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+        return actor;
     }
 
     @Override
     public Optional<Actor> get(Long id) {
-        return null;
+        Actor actor;
+        try (Session session = factory.openSession()) {
+            actor = session.get(Actor.class, id);
+        } catch (Exception e) {
+            throw new DataProcessingException(
+                    String.format("Can't get actor id = %s from DB", id), e);
+        }
+        return Optional.ofNullable(actor);
```

Comment:
```suggestion
        try (Session session = factory.openSession()) {
            return Optional.ofNullable(actor);
        } catch (Exception e) {
            throw new DataProcessingException(
                    String.format("Can't get actor id = %s from DB", id), e);
        }
```
the same for other dao implementations